packages/block-library/src/columns/edit.native.js

```diff
@@ -95,8 +105,11 @@ function ColumnsEditContainer( {
 			// show only 1 Column in row for mobile breakpoint container width
 			return 1;
 		} else if ( containerWidth < BREAKPOINTS.large ) {
-			// show 2 Column in row for large breakpoint container width
-			return Math.min( Math.max( 1, columnCount ), 2 );
+			// show LARGE_CONTAINER_COLUMNS_IN_ROW Column in row for large breakpoint container width
```

Comment:
Small nit: this comment actually is falsy, because when adding fresh Columns block it will show 2 columns instead of `LARGE_CONTAINER_COLUMNS_IN_ROW` for the large breakpoint.

Maybe rephrase it to: show the maximum number of columns in a row (up to max 3) for large breakpoint container width.